mlir/utils/jenkins/Jenkinsfile

```diff
@@ -802,7 +813,7 @@ pipeline {
                 axes {
                     axis {
                         name 'CHIP'
-                        values 'gfx908', 'gfx90a', 'gfx1030', 'gfx1100'
```

Comment:
More gfx1101.  If we tracked performance over time, we might see a discontinuity with this change.